### DISCUSSION TITLE
HasOneThrough displays a table similar to a HasMany relationship

### DISCUSSION THREAD

**Discussion by stephan-v** [laravel/nova-issues#5051] [Ideas & Feature Requests] [answered]
- Laravel Version: 9.38.0
- Nova Version: 4.18.2

### Description:

I don't think this is how it should work but having a HasOneThrough relationship in Laravel nova produces a table similar to a HasMany relationship. This is what is produces when you look at a resource which has this relationship set with `HasOneThrough::make('Plan')`

![image](https://user-images.githubusercontent.com/2384912/201337836-a250a27a-5be0-40a1-bf32-d90c81ea19ff.png)

Since only a single record will be returned from this relationship I expected to see something along the lines of this on the resource:

![image](https://user-images.githubusercontent.com/2384912/201337988-d8668d6c-e430-43de-8d7f-0ee28c09cbe1.png)

There is also no button to create a plan even though the policy clearly allows this (even when removing the policy entirely).

Am I misunderstanding something here because this does not seem to work at all.

### Detailed steps to reproduce the issue on a fresh Nova installation:

Just set up a `HasOneThrough::make()` on a Nova resource.

**Comment by crynobone** [comment, marked as answer]
Implemented in Nova 4.19.0

**Reply by jonnigg** [reply]
This says implemented in 4.19.0 but it seems that, in 4.34.3, HasOneThrough still shows a panel like a HasManyThrough or BelongsToMany would. Any chance you can elaborate on what "Implemented in Nova 4.19.0" means?

**Reply by crynobone** [reply]
If something is wrong here, you should instead create a new bug report with reproducing code so we can debug it further: https://github.com/laravel/nova-issues/issues/new?assignees=&labels=&projects=&template=1_Bug_report.md
